packages/contracts-bedrock/test/L1/DataAvailabilityChallenge.t.sol

```diff
@@ -150,27 +164,188 @@ contract DataAvailabilityChallenge_Withdraw_Test is DataAvailabilityChallenge_Te
         vm.expectRevert(abi.encodeWithSelector(IDataAvailabilityChallenge.WithdrawalFailed.selector));
         dataAvailabilityChallenge.withdraw();
     }
+
+    /// @notice Test that the `withdraw` function reverts with zero balance.
+    function test_withdraw_zeroBalance_reverts() public {
+        assertEq(dataAvailabilityChallenge.balances(address(this)), 0);
+
+        vm.expectRevert(abi.encodeWithSelector(IDataAvailabilityChallenge.WithdrawalFailed.selector));
+        dataAvailabilityChallenge.withdraw();
+    }
+}
```

Comment:
OK, this is interesting. There is no reason we would want to revert if someone attempts to withdraw a value of zero, it's pointless but they can do so if they wan. 

This test does pass (ie. the withdrawal does fail), but the reason for this is because the withdrawal is calling the fallback function on the test contract: 

<img width="3014" height="396" alt="CleanShot 2025-07-29 at 17 00 29@2x" src="https://github.com/user-attachments/assets/f18e15bb-db9e-462f-9d44-10c4578761e1" />

Since no fallback exists, it just reverts.

We can delete this test.